frontend/packages/kubevirt-plugin/package.json

```diff
@@ -3,12 +3,16 @@
   "version": "0.0.0-fixed",
   "description": "Kubevirt - Virtual machines addon for Kubernetes",
   "private": true,
+  "scripts": {
+    "lint": "yarn --cwd ../.. lintme packages/kubevirt-plugin"
+  },
   "dependencies": {
     "@console/internal": "0.0.0-fixed",
     "@console/plugin-sdk": "0.0.0-fixed",
     "@console/shared": "0.0.0-fixed",
     "kubevirt-web-ui-components": "~0.1.36"
   },
+
```

Comment:
Nit: please avoid newlines in `package.json` files.

Reply:
fixed